### DISCUSSION TITLE
New post type: Series

### DISCUSSION THREAD

**Discussion by OneOneSeven117** [stackernews/stacker.news#1878] [Ideas]
**problem**
Sometimes I'd like to subscribe to a certain recurring series of posts by stackers, without being notified about all of their other posts. 

**possible solution**
- A "series" post type allows stackers to select from their current series that they've started, which then brings up the template they've saved. The stacker then posts their next installment, and a new icon indicates the post is part of a series. 
- From the other perspective, this helps tighten up the notification system

**alter_natives** 
I thought about public series version as well, like the Parent Corner, where one person started and others join in. Maybe they are under a "Public" category but that could get crowded. A cost for starting a public series might help. 

**additional context**
additional coloring on the icon maybe to differentiate between:
- finished series
  -  Zap to Zero by @ek
- finite series
  -  Operation Saylor by @pillar
- recurring series
  - Meme Monday by @sn

**Comment by OneOneSeven117** [comment]
Some more thoughts from an SN Brainstorm: https://stacker.news/items/702478

**Comment by huumn** [comment]
Converting to discussion until I have better ideas on how to do this

**Reply by OneOneSeven117** [reply]
Curious if you've had any additional thoughts on this yet. This was something I wanted to tinker with in the dev environment.

**Reply by OneOneSeven117** [reply]
Or Instead of a new post type, it could work more like an additional option on any type of post, link/series, discussion/series, poll/series, bounty/series.

**Reply by huumn** [reply]
I think it might work best like so:

1. I go to the last post in the series
2. click the `...`
3. click `create next post in series`
4. then it opens up a post form maybe with some back links to all the prior posts in the series in the text area

Basically, I think we want to avoid another explicit post type and use the tools we already have to make it easier - creating a culture/pattern rather than another post button/"scheme" if that makes sense.

In service of this, we plan to show item references (e.g. If I reference `item #1` in `item #2` then when I view `item #1` I can see that `item #2` referenced it). That issue was #350 which we closed because we have it implemented but it's only used for notifications right now.

**Reply by OneOneSeven117** [reply]
>creating a culture/pattern rather than another post button/"scheme" if that makes sense.

Definitely. I realized after playing with the code that a new post type doesn't make much sense. 

>I go to the last post in the series
click the ...
click create next post in series

Could they still be subscribed to? 


>when I view item #1 I can see that item #2 referenced it

Would that be shown on the post or in details?

**Reply by huumn** [reply]
> Would that be shown on the post or in details?

Details to start. 

> Could they still be subscribed to?

I'd implement it by doing something to the effect of "if I'm subscribed to `item #1` and `item #2` references `item #1`, notify me of `item #2`" that way we can reuse existing item subscriptions and not need to create a special subscription.

-----------------------

With most of this stuff I like an iterative approach. Stage 1: minimal effort, minimal invasiveness. If Stage 1 proves out, then progress to Stage 2 levels of effort and invasiveness, and so on.
